Project/Fall2020_CSC403_Project/FrmDeath.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace Fall2020_CSC403_Project
+{
+    public partial class FrmDeath : Form
+    {
+        private Button Try_Again;
+        private Button Quit;
+        private Label label1;
```

Comment:
Maybe try adding a more specific name to this variable. What exactly does label1 display?

Reply:
label1 is just the text that I put on the death window. I didn't even realize that it auto-created all of these code.